rust/rust.bzl

```diff
@@ -49,6 +49,17 @@ rust_repositories()
   [Cargo](https://crates.io/).
 """
 
+# Reexporting some functions as part of our public api
+
+load(":bindgen.bzl", _rust_bindgen="rust_bindgen")
+
+rust_bindgen = _rust_bindgen
+""" See :bindgen.bzl for a complete description. """
+
+
+# @TODO Move the rest of the stuff in this file elsewhere..
```

Comment:
????

Reply:
Copying setup of https://github.com/bazelbuild/rules_go/blob/master/go/def.bzl , we probably want to move implementations out of this file and separately move some other stuff around (eg. toolchain.bzl is not at all self explanatory now that 'toolchain' is a bazel term, and should be split into toolchain.bzl and rustc.bzl or something similar)

(regardless i will remove this todo here)